masschallenge/impact-api
# [AC-4624b] - Permissions Revamp: Implement AccessControl object

**Changes Introduced:**
- add Clearance as defined in Accelerate.
- create parallel ClearanceFactory.
- register Clearance in the low-level API.
- add minimal test for Clearance to maintain coverage.

**Test:**
- in accelerate:
  - git checkout development;git pull;vagrant ssh;
  - resetdb;dumpdb
-  in impact api:
  - `make clean;make dev`
  - in a different terminal -
    - `make coverage` (see that 100% covered)
    -  `make dbload GZ_FILE=../accelerate/test_data/initial_schema.sql.gz`
  - login as demoadmin (or another superuser account)
  - see that Clearance is a suggested endpoint in the homepage.
  - Go to http://localhost:8000/api/impact/Clearance/ , see that the page loads successfully.
